Geometry/VeryForwardData/data/2021/v1/RP_220_Left_Station.xml

```diff
@@ -0,0 +1,154 @@
+    </UnionSolid>
+    <UnionSolid name="RP_220_Left_Station_Vacuum_5">
+      <rSolid name="RP_220_Left_Station_Vacuum_4"/>
+      <rSolid name="RP_220_Left_Station_Hor_Vacuum"/>
+      <rRotation name="RP_Transformations:RP_y_90_rot"/>
+      <Translation x="-(-[RP_220_Left_Hor_Vac_Length]/2+[RP_Device:RP_Device_Length_y]/2-[RP_Horizontal_Device:RP_Device_Hor_Closed_Wall_Thick_Int])" y="0*mm" z="-(-[RP_220_Left_Station_Length]/2+[RP_220_Left_Sec_Hor_z])"/>
+    </UnionSolid>
+  </SolidSection>
+  <LogicalPartSection label="RP_220_Left_Station.xml">
+    <LogicalPart name="RP_220_Left_Station">
+      <rSolid name="RP_220_Left_Station"/>
+      <rMaterial name="RP_Materials:Air"/>
+    </LogicalPart>
+    <LogicalPart name="RP_220_Left_Station_Tube_1">
+      <rSolid name="RP_220_Left_Station_Tube_1"/>
+      <rMaterial name="RP_Materials:AISI-316L-Steel"/>
+    </LogicalPart>
+    <LogicalPart name="RP_220_Left_Station_Tube_2">
+      <rSolid name="RP_220_Left_Station_Tube_2"/>
+      <rMaterial name="RP_Materials:AISI-316L-Steel"/>
+    </LogicalPart>
+    <LogicalPart name="RP_220_Left_Station_Tube_3">
+      <rSolid name="RP_220_Left_Station_Tube_3"/>
+      <rMaterial name="RP_Materials:AISI-316L-Steel"/>
+    </LogicalPart>
+    <LogicalPart name="RP_220_Left_Station_Tube_4">
+      <rSolid name="RP_220_Left_Station_Tube_4"/>
+      <rMaterial name="RP_Materials:AISI-316L-Steel"/>
+    </LogicalPart>
+    <LogicalPart name="RP_220_Left_Station_Tube_5">
+      <rSolid name="RP_220_Left_Station_Tube_5"/>
+      <rMaterial name="RP_Materials:AISI-316L-Steel"/>
+    </LogicalPart>
+    <LogicalPart name="RP_220_Left_Station_Vacuum_5">
+      <rSolid name="RP_220_Left_Station_Vacuum_5"/>
+      <rMaterial name="RP_Materials:RP_Primary_Vacuum"/>
+    </LogicalPart>
+  </LogicalPartSection>
+  <PosPartSection label="RP_220_Left_Station.xml">
+    <PosPart copyNumber="1">
+      <rParent name="RP_220_Left_Station"/>
+      <rChild name="RP_Vertical_Device:RP_Device_Vert_Corp_3"/>
+      <Translation x="0*mm" y="0*mm" z="-(-[RP_220_Left_Station_Length]/2+[RP_220_Left_Prim_Vert_z])"/>
+    </PosPart>
+    <PosPart copyNumber="2">
+      <rParent name="RP_220_Left_Station"/>
+      <rChild name="RP_Vertical_Device:RP_Device_Vert_Corp_3"/>
+      <Translation x="0*mm" y="0*mm" z="-(-[RP_220_Left_Station_Length]/2+[RP_220_Left_Sec_Vert_z])"/>
+    </PosPart>
+    <PosPart copyNumber="1">
+      <rParent name="RP_220_Left_Station"/>
+      <rChild name="RP_Horizontal_Device:RP_Device_Hor_Corp_3"/>
+      <Translation x="0*mm" y="0*mm" z="-(-[RP_220_Left_Station_Length]/2+[RP_220_Left_Prim_Hor_z])"/>
+    </PosPart>
+    <PosPart copyNumber="2">
+      <rParent name="RP_220_Left_Station"/>
+      <rChild name="RP_Horizontal_Device:RP_Device_Hor_Corp_3"/>
+      <Translation x="0*mm" y="0*mm" z="-(-[RP_220_Left_Station_Length]/2+[RP_220_Left_Sec_Hor_z])"/>
+    </PosPart>
+    <PosPart>
```

Comment:
As we found in #30292, all PosParts require copy numbers. Please fix all these XML files.